main.mk

```diff
@@ -58,7 +58,7 @@ DOCKER  ?= $(shell which docker)
 COMPOSE ?= $(shell which docker-compose)
 BUSYBOX_VERSION ?= 1.31.1
 
-JQ ?= $(DOCKER) run -i colstrom/jq
+JQ ?= $(DOCKER) run -i --rm colstrom/jq
 CUT ?= $(DOCKER) run -i busybox:$(BUSYBOX_VERSION) cut
```

Comment:
Should we also add --rm here and everywhere else?

Reply:
So I've fixed exactly that pain that  I've expirience by myself.